### DISCUSSION TITLE
Question about calculations on "cp2k/test/NNP/regtest-1"

### DISCUSSION THREAD

**Discussion by YuliangS** [cp2k/cp2k#3109] [Q&A]
Hi
I tried run calculations on the "cp2k/tests/NNP/regtest-1".   The results I got are close to the data in "TEST_FILES".
I am confused: Why the potential energy results for these calculations are positive?

**Comment by cschran** [comment]
Hi @YuliangS,

It's good that your tests are close to the reported values. 

The energies are positive because our cp2k implementation does not add the atomic energies back that were removed during training.
This is due to a mismatch between `RuNNer` and `n2p2` behaviour.
If you add the keyword `remove_atom_energies` to the `input.nn` file, `cp2k` will add them back after the evaluation of the NNPs. This keyword is used in `RuNNer`, but was removed from `n2p2`.
Hope this helps explain your observations.
Best,
Christoph

**Comment by YuliangS** [comment]
I got it.

Thank you so much for the explanation
